spring-integration-file/src/test/java/org/springframework/integration/file/config/FileTailInboundChannelAdapterParserTests.java

```diff
@@ -95,6 +95,7 @@ public void testNative() {
 		assertSame(exec, TestUtils.getPropertyValue(nativeAdapter, "taskExecutor"));
 		assertSame(sched, TestUtils.getPropertyValue(nativeAdapter, "taskScheduler"));
 		assertTrue(TestUtils.getPropertyValue(nativeAdapter, "autoStartup", Boolean.class));
+		assertTrue(TestUtils.getPropertyValue(nativeAdapter, "enableStatusReader" ,Boolean.class));
```

Comment:
OK. 
Now let's make make a test for `assertFalse()`.
I mean we should be sure that changes to the parser reflects to the target `OSDelegatingFileTailingMessageProducer`.

What I want to say that it isn't possible now.
You should add `enable-status-reader` XSD attribute and modify `FileTailInboundChannelAdapterParserTests-context.xml` with `enable-status-reader="false"`